src/components/popup/intro/intro.less

```diff
@@ -10,7 +10,7 @@
 
 div.intro {
 	align-items: center;
-	padding: 0 4em;
+	/*padding: 0 4em;*/
```

Comment:
This should probably be deleted

Reply:
The version you've made this comment on seems outdated. I can't see the comment in the files changed view...